### ISSUE TITLE
Interface to find Slugs

### ISSUE THREAD

**Issue by pearkes** [digitalocean/api-v2#71]
It'd be great to see what the slugs were from some web page.

See https://github.com/hashicorp/terraform/issues/143 for more.

**Comment by brookemckim** [comment]
Agreed, will get this added to the developers.digitalocean.com page.

**Comment by phillbaker** [comment]
The "Response Body" under https://developers.digitalocean.com/#list-all-sizes now lists all of the available sizes, however, it's a bit hard to parse as its all the objects. This doesn't list all of the available distributions either. Just wanted to offer an update.
